handlers/user.go

```diff
@@ -212,7 +212,7 @@ func UserAuthorizeConfirm(w http.ResponseWriter, r *http.Request) {
 
 	err = authorizeTemplate.ExecuteTemplate(w, "layout", data)
 	if err != nil {
-		logger.Errorf("error executing template for %v route: %v", r.URL.String(), err)
+		logger.Errorf("error executing template [user.go / UserAuthorizeConfirm] for %v route: %v", r.URL.String(), err)
```

Comment:
Should print "()" instead of "[]".

Reply:
It's on purpose, as it's not using handleTemplateError